packages/build/src/install/missing.js

```diff
@@ -21,6 +21,18 @@ export const installMissingPlugins = async function ({ missingPlugins, autoPlugi
   await addExactDependencies({ packageRoot: autoPluginsDir, isLocal: mode !== 'buildbot', packages })
 }
 
+export const installIntegrationPlugins = async function ({ integrations, autoPluginsDir, mode, logs }) {
+  const packages = integrations.map(getIntegrationPackage)
+  logInstallIntegrations(logs, integrations)
+
+  await createAutoPluginsDir(logs, autoPluginsDir)
+  await addExactDependencies({ packageRoot: autoPluginsDir, isLocal: mode !== 'buildbot', packages })
+}
+
+const getIntegrationPackage = function ({ version }) {
+  return `${version}/packages/buildhooks.tgz`
```

Comment:
More for my own curiosity - is the `version` just the git commit hash or does this include the whole site URL as well?

Reply:
whole deploy URL is returned from `jigsaw`.

A hash of some form will be on the `version` inside the generated buildhooks for cache busting purposes